pages/index.vue

```diff
@@ -101,11 +101,9 @@
         </v-card>
       </v-col>
       <v-col cols="12" md="4">
-        <ul v-if="loaded" style="list-style-type:none">
-          <li>
-            <h3 class="h2 mt-2 mb-2">{{ $t('home.news.title') }}</h3>
-          </li>
-          <li v-for="(newsItem, i) in recentNewsItems" :key="i">
+        <h3 class="h2 mt-2 mb-2">{{ $t('home.news.title') }}</h3>
+        <span v-if="loaded">
```

Comment:
Switch to a `<div>` instead, since this is a block level. Reserve the use of `<span>` for inline HTML like in a `<p>` or `<h2>`.